.gitignore

```diff
@@ -25,3 +25,4 @@ src/test/data/sandbox/
 # MacOS custom attributes files created by Finder
 .DS_Store
 docs/_site/
+gradle/wrapper/gradle-wrapper.properties
```

Comment:
Think this ignore is not needed.

Reply:
My local machine uses gradle v6.3, which fails on merges with master for that is dependent on gradle v5.2.1. Still trying to find a workaround.

Reply:
we can bump the Gradle version if that helps?